tests/conftest.py

```diff
@@ -160,6 +160,15 @@ def pytest_addoption(parser):
         action="store_true",
         help="Generate new files into __snapshot__ directories instead of checking against existing files",
     )
+    parser.addoption("--slow", action="store_true", help="include tests marked slow")
```

Comment:
`--include-slow/--no-include-slow` seems a bit more in-keeping with clig.dev guidelines, but I am not so familiar with conventions around pytest

Reply:
how about `--with-slow` ? I think `includes` is a mildly special word ( https://docs.pytest.org/en/6.2.x/reference.html?highlight=include#pytest-freeze-includes ), and `--with-slow` is maybe more conversational? "run pytest with slow tests," which matches poetry's `--with` 

no strong feelings, good with either, prefer both to just `--slow`, your call :)

Reply:
I like `--with-slow`. And I've come to peace with adding an custom option 😂 Whenever you have a chance to make the change I'm happy to merge.